test/TestApp.AspNetCore/AssemblyInfo.cs

```diff
@@ -14,11 +14,10 @@
 // limitations under the License.
 // </copyright>
 
-using System.Diagnostics.CodeAnalysis;
+using System.Runtime.CompilerServices;
 
-[assembly: SuppressMessage(
-    "StyleCop.CSharp.NamingRules",
-    "SA1300",
-    Justification = "Reviewed.",
-    Scope = "namespaceanddescendants",
-    Target = "TestApp.AspNetCore._7._0")]
+#if SIGNED
+[assembly: InternalsVisibleTo("OpenTelemetry.Instrumentation.AspNetCore.Tests, PublicKey=002400000480000094000000060200000024000052534131000400000100010051c1562a090fb0c9f391012a32198b5e5d9a60e9b80fa2d7b434c9e5ccb7259bd606e66f9660676afc6692b8cdc6793d190904551d2103b7b22fa636dcbb8208839785ba402ea08fc00c8f1500ccef28bbf599aa64ffb1e1d5dc1bf3420a3777badfe697856e9d52070a50c3ea5821c80bef17ca3acffa28f89dd413f096f898")]
```

Comment:
why we need this trick? everything needed can be public in the test app

Reply:
This is not needed. Removed it.